<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="" />


<title>ATACseq_analysis.knit</title>

<script src="site_libs/header-attrs-2.18/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="site_libs/reactable-binding-0.3.0/reactable.js"></script>
<script src="site_libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.11.1/plotly-latest.min.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">10T K36me2</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">K36me2 summary</a>
</li>
<li>
  <a href="QKO_analysis.html">QKO reps analysis</a>
</li>
<li>
  <a href="ATACseq_analysis.html">ATAC-seq summary</a>
</li>
<li>
  <a href="differential_atac.html">Differential ATAC</a>
</li>
<li>
  <a href="HOMER_output.html">Homer results</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="number-of-peaks-per-sample-idr-pooled" class="section level1">
<h1>Number of peaks per sample (IDR-pooled)</h1>
<div id="htmlwidget-0c2bd6afe19cb2ad13c7" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0c2bd6afe19cb2ad13c7">{"x":{"tag":{"name":"Reactable","attribs":{"data":{".rownames":["Number_of_Peaks"],"PA":[32305],"TKO":[33323],"QKO":[26636]},"columns":[{"accessor":".rownames","name":"","type":"character","sortable":false,"filterable":false,"rowHeader":true},{"accessor":"PA","name":"PA","type":"numeric"},{"accessor":"TKO","name":"TKO","type":"numeric"},{"accessor":"QKO","name":"QKO","type":"numeric"}],"defaultPageSize":10,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"b5268781a38940cfdee8c04857370817"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="aggregate-and-heatmap-plots" class="section level1 tabset">
<h1 class="tabset">Aggregate and heatmap plots</h1>
<p>ATAC-seq signal was centered on ATAC-seq peaks.</p>
<div id="merged-reps" class="section level2">
<h2>Merged reps</h2>
<p><img src="images/ATAC.merged_reps.heatmap.png" width="50%" /></p>
</div>
<div id="individual-reps" class="section level2">
<h2>Individual reps</h2>
<p><img src="images/ATAC.individual_reps.heatmap.png" width="50%" /></p>
</div>
</div>
<div id="distribution-of-atac-seq-peaks" class="section level1">
<h1>Distribution of ATAC-seq peaks</h1>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-2-1.png" width="100%" /></p>
</div>
<div id="pca" class="section level1">
<h1>PCA</h1>
<div id="htmlwidget-c4e4d5f44798dc97fdc5" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-c4e4d5f44798dc97fdc5">{"x":{"visdat":{"14c07771d6e68":["function () ","plotlyVisDat"]},"cur_data":"14c07771d6e68","attrs":{"14c07771d6e68":{"x":[-31.8397341200494,-31.9209003060315,17.1032533787394,19.5423798794809,14.3516903782002,12.7633107896604],"y":[0.732723885123169,0.619579426697174,6.69953144345515,6.17921901977061,-6.67208287193785,-7.55897090310827],"text":["parental-1","parental-2","TKO-1","TKO-dgNSD3_c10_unedited","QKO_c1","QKO_c7"],"mode":"markers","marker":{"size":11},"color":["PA","PA","TKO","TKO","QKO","QKO"],"colors":"Dark2","alpha_stroke":1,"sizes":[10,100],"spans":[1,20]}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"","xaxis":{"domain":[0,1],"automargin":true,"title":"PC1 ( 68.54%)"},"yaxis":{"domain":[0,1],"automargin":true,"title":"PC2 ( 16.84%)"},"hovermode":"closest","showlegend":true},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"x":[-31.8397341200494,-31.9209003060315],"y":[0.732723885123169,0.619579426697174],"text":["parental-1","parental-2"],"mode":"markers","marker":{"color":"rgba(27,158,119,1)","size":11,"line":{"color":"rgba(27,158,119,1)"}},"type":"scatter","name":"PA","textfont":{"color":"rgba(27,158,119,1)"},"error_y":{"color":"rgba(27,158,119,1)"},"error_x":{"color":"rgba(27,158,119,1)"},"line":{"color":"rgba(27,158,119,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[14.3516903782002,12.7633107896604],"y":[-6.67208287193785,-7.55897090310827],"text":["QKO_c1","QKO_c7"],"mode":"markers","marker":{"color":"rgba(217,95,2,1)","size":11,"line":{"color":"rgba(217,95,2,1)"}},"type":"scatter","name":"QKO","textfont":{"color":"rgba(217,95,2,1)"},"error_y":{"color":"rgba(217,95,2,1)"},"error_x":{"color":"rgba(217,95,2,1)"},"line":{"color":"rgba(217,95,2,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[17.1032533787394,19.5423798794809],"y":[6.69953144345515,6.17921901977061],"text":["TKO-1","TKO-dgNSD3_c10_unedited"],"mode":"markers","marker":{"color":"rgba(117,112,179,1)","size":11,"line":{"color":"rgba(117,112,179,1)"}},"type":"scatter","name":"TKO","textfont":{"color":"rgba(117,112,179,1)"},"error_y":{"color":"rgba(117,112,179,1)"},"error_x":{"color":"rgba(117,112,179,1)"},"line":{"color":"rgba(117,112,179,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="volcano-plots" class="section level1 tabset">
<h1 class="tabset">Volcano plots</h1>
<p>These logfold changes were shrunken prior to plotting.</p>
<div id="tko-v-pa" class="section level2 tabset">
<h2 class="tabset">TKO v PA</h2>
<div id="log2fc-1" class="section level3">
<h3>|log2FC| &gt; 1</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-4-1.png" width="100%" /></p>
</div>
<div id="log2fc-2" class="section level3">
<h3>|log2FC| &gt; 2</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-5-1.png" width="100%" /></p>
</div>
<div id="log2fc-2.5" class="section level3">
<h3>|log2FC| &gt; 2.5</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-6-1.png" width="100%" /></p>
</div>
<div id="log2fc-3" class="section level3">
<h3>|log2FC| &gt; 3</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-7-1.png" width="100%" /></p>
</div>
<div id="log2fc-4" class="section level3">
<h3>|log2FC| &gt; 4</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-8-1.png" width="100%" /></p>
</div>
</div>
<div id="qko-v-pa" class="section level2 tabset">
<h2 class="tabset">QKO v PA</h2>
<div id="log2fc-1-1" class="section level3">
<h3>|log2FC| &gt; 1</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-9-1.png" width="100%" /></p>
</div>
<div id="log2fc-2-1" class="section level3">
<h3>|log2FC| &gt; 2</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-10-1.png" width="100%" /></p>
</div>
<div id="log2fc-2.5-1" class="section level3">
<h3>|log2FC| &gt; 2.5</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-11-1.png" width="100%" /></p>
</div>
<div id="log2fc-3-1" class="section level3">
<h3>|log2FC| &gt; 3</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-12-1.png" width="100%" /></p>
</div>
<div id="log2fc-4-1" class="section level3">
<h3>|log2FC| &gt; 4</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-13-1.png" width="100%" /></p>
</div>
</div>
<div id="qko-v-tko" class="section level2 tabset">
<h2 class="tabset">QKO v TKO</h2>
<div id="log2fc-1-2" class="section level3">
<h3>|log2FC| &gt; 1</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-14-1.png" width="100%" /></p>
</div>
<div id="log2fc-2-2" class="section level3">
<h3>|log2FC| &gt; 2</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-15-1.png" width="100%" /></p>
</div>
<div id="log2fc-2.5-2" class="section level3">
<h3>|log2FC| &gt; 2.5</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-16-1.png" width="100%" /></p>
</div>
<div id="log2fc-3-2" class="section level3">
<h3>|log2FC| &gt; 3</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-17-1.png" width="100%" /></p>
</div>
<div id="log2fc-4-2" class="section level3">
<h3>|log2FC| &gt; 4</h3>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-18-1.png" width="100%" /></p>
</div>
</div>
</div>
<div id="ma-plots" class="section level1 tabset">
<h1 class="tabset">MA plots</h1>
<p>Points will be colored blue if the adjusted p value is less than
0.05. Points which fall out of the window are plotted as open triangles
pointing either up or down.</p>
<div id="tko-v-pa-1" class="section level2">
<h2>TKO v PA</h2>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-19-1.png" width="100%" /></p>
</div>
<div id="qko-v-pa-1" class="section level2">
<h2>QKO v PA</h2>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-20-1.png" width="100%" /></p>
</div>
<div id="qko-v-tko-1" class="section level2">
<h2>QKO v TKO</h2>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-21-1.png" width="100%" /></p>
</div>
</div>
<div id="feature-distribution" class="section level1 tabset">
<h1 class="tabset">Feature distribution</h1>
<p>For all differential peaks, padj &lt; 0.05 was used with different
log2FC as indicates below.</p>
<div id="tko-v-pa-2" class="section level2 tabset">
<h2 class="tabset">TKO v PA</h2>
<div id="cutoff1" class="section level3 tabset">
<h3 class="tabset">Cutoff=1</h3>
<div id="log2fc-1-3" class="section level4">
<h4>log2FC &gt; 1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:23 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:24 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:24 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:24 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:02:44 
## &gt;&gt; done...                    2022-11-29 14:02:44</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-22-1.png" width="100%" /></p>
</div>
<div id="log2fc--1" class="section level4">
<h4>log2FC &lt; -1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:45 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:45 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:45 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:45 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:02:47 
## &gt;&gt; done...                    2022-11-29 14:02:47</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-23-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff2" class="section level3 tabset">
<h3 class="tabset">Cutoff=2</h3>
<div id="log2fc-2-3" class="section level4">
<h4>log2FC &gt; 2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:49 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:49 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:49 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:49 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:02:51 
## &gt;&gt; done...                    2022-11-29 14:02:51</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-24-1.png" width="100%" /></p>
</div>
<div id="log2fc--2" class="section level4">
<h4>log2FC &lt; -2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:52 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:52 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:52 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:52 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:02:55 
## &gt;&gt; done...                    2022-11-29 14:02:55</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-25-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff3" class="section level3 tabset">
<h3 class="tabset">Cutoff=3</h3>
<div id="log2fc-3-3" class="section level4">
<h4>log2FC &gt; 3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:55 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:55 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:56 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:56 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:02:58 
## &gt;&gt; done...                    2022-11-29 14:02:58</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-26-1.png" width="100%" /></p>
</div>
<div id="log2fc--3" class="section level4">
<h4>log2FC &lt; -3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:02:59 
## &gt;&gt; identifying nearest features...        2022-11-29 14:02:59 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:02:59 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:02:59 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:01 
## &gt;&gt; done...                    2022-11-29 14:03:01</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-27-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff4" class="section level3 tabset">
<h3 class="tabset">Cutoff=4</h3>
<div id="log2fc-4-3" class="section level4">
<h4>log2FC &gt; 4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:02 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:02 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:02 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:02 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:04 
## &gt;&gt; done...                    2022-11-29 14:03:04</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-28-1.png" width="100%" /></p>
</div>
<div id="log2fc--4" class="section level4">
<h4>log2FC &lt; -4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:05 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:05 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:06 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:06 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:08 
## &gt;&gt; done...                    2022-11-29 14:03:08</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-29-1.png" width="100%" /></p>
</div>
</div>
</div>
<div id="qko-v-pa-2" class="section level2 tabset">
<h2 class="tabset">QKO v PA</h2>
<div id="cutoff1-1" class="section level3 tabset">
<h3 class="tabset">Cutoff=1</h3>
<div id="log2fc-1-4" class="section level4">
<h4>log2FC &gt; 1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:09 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:09 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:09 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:09 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:11 
## &gt;&gt; done...                    2022-11-29 14:03:11</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-30-1.png" width="100%" /></p>
</div>
<div id="log2fc--1-1" class="section level4">
<h4>log2FC &lt; -1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:13 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:13 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:13 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:13 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:15 
## &gt;&gt; done...                    2022-11-29 14:03:15</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-31-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff2-1" class="section level3 tabset">
<h3 class="tabset">Cutoff=2</h3>
<div id="log2fc-2-4" class="section level4">
<h4>log2FC &gt; 2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:16 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:16 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:16 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:16 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:18 
## &gt;&gt; done...                    2022-11-29 14:03:18</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-32-1.png" width="100%" /></p>
</div>
<div id="log2fc--2-1" class="section level4">
<h4>log2FC &lt; -2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:19 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:19 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:19 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:19 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:21 
## &gt;&gt; done...                    2022-11-29 14:03:21</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-33-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff3-1" class="section level3 tabset">
<h3 class="tabset">Cutoff=3</h3>
<div id="log2fc-3-4" class="section level4">
<h4>log2FC &gt; 3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:22 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:22 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:23 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:23 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:25 
## &gt;&gt; done...                    2022-11-29 14:03:25</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-34-1.png" width="100%" /></p>
</div>
<div id="log2fc--3-1" class="section level4">
<h4>log2FC &lt; -3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:26 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:26 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:26 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:26 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:28 
## &gt;&gt; done...                    2022-11-29 14:03:28</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-35-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff4-1" class="section level3 tabset">
<h3 class="tabset">Cutoff=4</h3>
<div id="log2fc-4-4" class="section level4">
<h4>log2FC &gt; 4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:29 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:29 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:30 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:30 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:32 
## &gt;&gt; done...                    2022-11-29 14:03:32</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-36-1.png" width="100%" /></p>
</div>
<div id="log2fc--4-1" class="section level4">
<h4>log2FC &lt; -4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:32 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:32 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:33 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:33 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:35 
## &gt;&gt; done...                    2022-11-29 14:03:35</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-37-1.png" width="100%" /></p>
</div>
</div>
</div>
<div id="qko-v-tko-2" class="section level2 tabset">
<h2 class="tabset">QKO v TKO</h2>
<div id="cutoff1-2" class="section level3 tabset">
<h3 class="tabset">Cutoff=1</h3>
<div id="log2fc-1-5" class="section level4">
<h4>log2FC &gt; 1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:36 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:36 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:36 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:36 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:38 
## &gt;&gt; done...                    2022-11-29 14:03:38</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-38-1.png" width="100%" /></p>
</div>
<div id="log2fc--1-2" class="section level4">
<h4>log2FC &lt; -1</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:40 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:40 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:40 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:40 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:42 
## &gt;&gt; done...                    2022-11-29 14:03:42</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-39-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff2-2" class="section level3 tabset">
<h3 class="tabset">Cutoff=2</h3>
<div id="log2fc-2-5" class="section level4">
<h4>log2FC &gt; 2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:44 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:44 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:44 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:44 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:46 
## &gt;&gt; done...                    2022-11-29 14:03:46</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-40-1.png" width="100%" /></p>
</div>
<div id="log2fc--2-2" class="section level4">
<h4>log2FC &lt; -2</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:47 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:47 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:47 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:47 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:50 
## &gt;&gt; done...                    2022-11-29 14:03:50</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-41-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff3-2" class="section level3 tabset">
<h3 class="tabset">Cutoff=3</h3>
<div id="log2fc-3-5" class="section level4">
<h4>log2FC &gt; 3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:51 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:51 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:51 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:51 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:53 
## &gt;&gt; done...                    2022-11-29 14:03:53</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-42-1.png" width="100%" /></p>
</div>
<div id="log2fc--3-2" class="section level4">
<h4>log2FC &lt; -3</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:55 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:55 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:55 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:55 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:03:57 
## &gt;&gt; done...                    2022-11-29 14:03:57</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-43-1.png" width="100%" /></p>
</div>
</div>
<div id="cutoff4-2" class="section level3 tabset">
<h3 class="tabset">Cutoff=4</h3>
<div id="log2fc-4-5" class="section level4">
<h4>log2FC &gt; 4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:03:58 
## &gt;&gt; identifying nearest features...        2022-11-29 14:03:58 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:03:59 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:03:59 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:04:01 
## &gt;&gt; done...                    2022-11-29 14:04:01</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-44-1.png" width="100%" /></p>
</div>
<div id="log2fc--4-2" class="section level4">
<h4>log2FC &lt; -4</h4>
<pre><code>## &gt;&gt; preparing features information...      2022-11-29 14:04:02 
## &gt;&gt; identifying nearest features...        2022-11-29 14:04:02 
## &gt;&gt; calculating distance from peak to TSS...   2022-11-29 14:04:02 
## &gt;&gt; assigning genomic annotation...        2022-11-29 14:04:02 
## &gt;&gt; assigning chromosome lengths           2022-11-29 14:04:05 
## &gt;&gt; done...                    2022-11-29 14:04:05</code></pre>
<p><img src="ATACseq_analysis_files/figure-html/unnamed-chunk-45-1.png" width="100%" /></p>
</div>
</div>
</div>
</div>
<div id="number-of-atacseq-peaks-in-k36me2-regions"
class="section level1">
<h1>Number of ATACseq peaks in K36me2 regions</h1>
<p>Taking the union of ATACseq, how many ATACseq peaks are found in each
sample’s K36me2 regions.</p>
<div id="htmlwidget-c0f585790470d5322905" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c0f585790470d5322905">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"PA":[34636],"PA_only":[16467],"TKO":[23228],"TKO_only":[3728]},"columns":[{"accessor":"PA","name":"PA","type":"numeric"},{"accessor":"PA_only","name":"PA_only","type":"numeric"},{"accessor":"TKO","name":"TKO","type":"numeric"},{"accessor":"TKO_only","name":"TKO_only","type":"numeric"}],"defaultPageSize":10,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"7fdd69825e3d997ae7a77e1a0138296e"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
